scripts/vmpublish.rb

```diff
@@ -0,0 +1,80 @@
+require 'logger'
+require 'pathname'
+require 'trollop'
+require 'json'
+
+require_relative 'config'
+
+
```

Comment:
This extra whitespace here indicates in my mind that the methods below belong in a namespace, such as `Build` if publishing is a step in the build process.